test/mbr_update_test.rb

```diff
@@ -28,6 +31,16 @@
       )
     end
 
+    context "on s390" do
+      before do
+        allow(Yast::Arch).to receive(:architecture).and_return("s390_64")
+      end
+
+      it "does nothing except returning true" do
+        expect(subject.run).to eq true
```

Comment:
@arvidjaar Yes. Copy&paste error.

@jreidinger Yes, it's kind of annoying. But otherwise you are actually not checking that s390 is actually skipped. Maybe there is a better way, but the current one (just relying on the absence of mocks for the testsuite to crash) doesn't look correct to me.